lib/src/main/kotlin/com/swmansion/starknet/data/types/conversions/ConvertibleToFeltList.kt

```diff
@@ -0,0 +1,14 @@
+package com.swmansion.starknet.data.types.conversions
+
+import com.swmansion.starknet.data.types.Felt
+
+/**
+ * Implementers of this interface support conversions to
+ * the list of Felts.
+ */
+interface ConvertibleToFeltList {
```

Comment:
Felt itself should support this interface. This way you will be able to mix felts with other data structures.

Reply:
And how exactly that would work? Converting a felt to a list with single felt doesn't make sense at all. A user can just do `listOf(Felt)`.

Reply:
It would return a list with a single felt. Without Felt supporting it there is no point in doing that at all. The whole reason why I want this interface is to be able to do `calldata=[Felt.ONE, Uint256(somthing), otherStrucutre]` that would be serialised by sdk.